File: src/main/java/timberforest/app/Application.java
package timberforest.app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@Configuration
@EnableAutoConfiguration
@ComponentScan("timberforest.app")
public class Application {
    public static void main(String[] args) throws Exception {
        // AnsiOutput.setEnabled(Enabled.ALWAYS);
        SpringApplication.run(Application.class, args);
    }
}

File: src/main/kotlin/timberforest/app/Application.kt
package timberforest.app

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}

/**
 * Created by masahiro on 2016/04/14.
 */
@Configuration
@EnableAutoConfiguration
@ComponentScan("timberforest.app")
@SpringBootApplication
open class Application {
}
